dpub-aam/dpub-aam.html

```diff
@@ -437,7 +437,7 @@
                                                                 </td>
                                                                 <td><p>Expose <code>ROLE_SECTION</code> and object attribute <code>xml-roles:doc-colophon</code>. </p></td>
                                                                 <td>AXRole: <code>AXGroup</code><br />
-                                                                        AXSubrole: <code>&lt;nil&gt;</code><br/>
+                                                                        AXSubrole: AXApplicationGroup</code><br/>
```

Comment:
These are probably right, but the specific role being changed is outside the diff lines, so it's difficult to be certain from this view.